pom.xml

```diff
@@ -234,14 +234,29 @@
         </exclusion>
       </exclusions>
     </dependency>
-
     <!-- static analysis -->
     <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
       <scope>provided</scope>
       <optional>true</optional>
     </dependency>
+
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>bouncycastle-api</artifactId>
+      <version>2.29</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jackson2-api</artifactId>
+      <version>2.15.2-350.v0c2f3f8fc595</version>
+    </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>okhttp-api</artifactId>
+      <version>4.11.0-157.v6852a_a_fa_ec11</version>
+    </dependency>
```

Comment:
I do not depend on these and so they should not be needed.   What is the rationale for the the creep in dependencies?

perhaps you just need to pickup a newer bom version?